src/node_platform.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_NODE_PLATFORM_H_
+#define SRC_NODE_PLATFORM_H_
+
+#include "libplatform/libplatform.h"
+#include "node_mutex.h"
+#include "uv.h"
+
+#include <queue>
+
+namespace node {
+
+using v8::Isolate;
+using v8::Platform;
+using v8::Task;
+
+class NodePlatform : public Platform {
+ public:
+  NodePlatform(int thread_pool_size, uv_loop_t* loop,
+               v8::TracingController* tracing_controller);
```

Comment:
Would be nice if the API for NodePlatform did not depend on v8 specific types like v8::TracingController and v8::Isolate (below) but it is still a good start just be abstracting NodePlatform itself.